src/tests/multihost/admultidomain/test_multiforest.py

```diff
@@ -11,61 +11,59 @@
 class TestADMultiForest(object):
 
     @staticmethod
-    def test_0001_multiforest(multihost, newhostname, adjoin):
+    def test_0001_multiforest(self, multihost, newhostname, adjoin):
         """
-        :title: IDM-SSSD-TC: ad_provider: admultiforest
-        :id:
+        :title: IDM-SSSD-TC: ad_provider: admultiforest : This test configures SSSD to contain two domain entries
+        in two different forests. This is a hack, but has a valid customer use case to be tested.
```

Comment:
'This is a hack, but has a valid customer use case to be tested.',
can we add this as a comment?
And also can you rephrase the Title.